src/etc/gdb_load_rust_pretty_printers.py

```diff
@@ -9,4 +9,4 @@
 # except according to those terms.
 
 import gdb_rust_pretty_printing
-gdb_rust_pretty_printing.register_printers(gdb.current_objfile())
+gdb_rust_pretty_printing.register_printers(gdb)
```

Comment:
@michaelwoerister,  After really thinking from what you mention and the code itself, I see what you mean. So I need to find another way to make the code work for python 2.7.13 & gdb 7.12.1. 

The reason for this change because it cause a problem where `gdb.current_objfile()` would return `None` unless this file is auto-loaded for gdb. 

I'll place an open issue if I'm not able to figure out a workaround. If comes to that, should I remove this PR?